app/assets/stylesheets/style.css.scss

```diff
@@ -56,3 +56,8 @@ footer.page-footer {
 .full-width {
     width: 100%;
 }
+
+.bg-text{
+  min-width: 800px; min-height: 600px;
```

Comment:
Properties should be ordered background, min-height, min-width<br>Property 'min-height' should be placed on own line